modules/moa-installing-prerequisites.adoc

```diff
@@ -0,0 +1,216 @@
+// Module included in the following assemblies:
+//
+// getting_started_moa/creating-first-moa-cluster.adoc
+
+
+[id="moa-installing-prerequisites"]
+= Installing prerequisites
+
+Complete the following prerequisites before creating your {product-title} cluster.
+
+.Procedure
+
+. Select an AWS account to use
++
+Unless your just testing out MOA, we recommend using a dedicated AWS account to run any production clusters. If you are utilizing AWS Organizations, you can use an AWS account within your organization or link:https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_accounts_create.html#orgs_manage_accounts_create-new[create a new one].
++
+If you are using AWS organizations and you need to have a Service Control Policy (SCP) applied to the AWS account you plan to use, see the link:https://www.openshift.com/dedicated/ccs#scp[Red Hat Requirements for Customer Cloud Subscriptions] for details on the minimum required SCP.
++
+As part of the cluster creation process, `moactl` will perform the following actions:
```

Comment:
The way it's currently structured there is only one action listed, creating the osdcCcsAdmin IAM user.  And once this is done, by default the user inherits specific permissions and policies? I would remove the bulleted list here, unless you plan to add more actions.

"As part of the cluster creation process, moactl will create an osdCcsAdmin IAM user. This user will leverage the IAM credentials you provide when configuring the AWS cli in the next section.
Note: This user will have Programmatic access enabled and the AdministratorAccess policy attached to it."

Reply:
Great catch. Yes, we used to have more actions but decided spelling them all out here was not a good choice.